.github/workflows/c8run-release.yaml

```diff
@@ -46,24 +46,17 @@ jobs:
     name: Create C8Run tag/release
     runs-on: ubuntu-latest
     timeout-minutes: 15
-    outputs:
-      NEXUS_USERNAME: ${{ steps.secrets.outputs.NEXUS_PASSWORD }}
-      NEXUS_PASSWORD: ${{ steps.secrets.outputs.NEXUS_PASSWORD }}
     steps:
+      - name: ℹ️ Print workflow inputs ℹ️
+        shell: bash
+        env:
+          GITHUB_CONTEXT: ${{ toJson(inputs) }}
```

Comment:
🔧 Let's avoid declaring environment variables starting with the `GITHUB_` prefix which might clash if GH ever introduces such a variable on their own. Just calling this `WORKFLOW_INPUTS: ${{ toJson(inputs) }}` should work as well.

Reply:
I renamed this variable. thanks